docs/python_support/reference.md

```diff
@@ -0,0 +1,769 @@
+
+```pbtxt
+node {
+  name: "python_node"
+  calculator: "PythonExecutorCalculator"
+  input_side_packet: "PYTHON_NODE_RESOURCES:py"
+  input_stream: "INPUT:input"
+  input_stream: "LOOPBACK:loopback"
+  input_stream_info: {
+    tag_index: 'LOOPBACK:0',
+    back_edge: true
+  }
+  input_stream_handler {
+    input_stream_handler: "SyncSetInputStreamHandler",
+    options {
+      [mediapipe.SyncSetInputStreamHandlerOptions.ext] {
+        sync_set {
+          tag_index: "LOOPBACK:0"
+        }
+      }
+    }
+  }
+  output_stream: "OUTPUT:output"
+  output_stream: "LOOPBACK:loopback"
+  node_options: {
+    [type.googleapis.com / mediapipe.PythonExecutorCalculatorOptions]: {
+      handler_path: "/ovms/workspace/model.py"
+    }
+  }
+}
+```
+
+Apart from basic configuration present also in regular mode, this graph contains some additional content. Let's review it.
+
+1. `LOOPBACK` input and output stream
+
+    ```
+    input_stream: "LOOPBACK:loopback"
+    ...
+    output_stream: "LOOPBACK:loopback"
+    ```
+
+    This set of additional input and output stream enables internal cycle inside the node. It is used to trigger `Process` calls without any incoming packets and therefore call the generator without new data. The value in both input and output stream must be exactly the same and the `PythonExecutorCalculator` always expects the tag to be `LOOPBACK`.
+
+    `LOOPBACK` input is not passed to `execute` method.
+
+2. Back Edge Annotation
+    ```
+    input_stream_info: {
+      tag_index: 'LOOPBACK:0',
+      back_edge: true
+    }
+    ```
+
+    This part says that the input stream with tag `LOOPBACK` and index `0` is used to create a cycle. If there are more than one index for `LOOPBACK` tag, the `PythonExecutorCalculator` will ignore it.
+
+3. `SyncSetInputStreamHandler`
+
+    ```
+    input_stream_handler {
+      input_stream_handler: "SyncSetInputStreamHandler",
+      options {
+        [mediapipe.SyncSetInputStreamHandlerOptions.ext] {
+          sync_set {
+            tag_index: "LOOPBACK:0"
+          }
+        }
+      }
+    }
+    ```
+    
+    In regular configuration `DefaultInputStreamHandler` is used, but for generative mode it's not sufficient. When default handler is defined, node waits for all input streams before calling `Process`. In generative mode `Process` should be called once for data coming from the graph and then multiple times only by receiving signal on `LOOPBACK`, but inputs from a graph and `LOOPBACK` will never be present at the same time. 
+
+    For generative mode to work, inputs from the graph and `LOOPBACK` must be decoupled, meaning `Process` can be called with complete set of inputs from the graph, but also with just `LOOPBACK`. It can be acheived via `SyncSetInputStreamHandler`. Above configuration sample creates a set with `LOOPBACK`, which also, implicitly creates another set with all remaining  inputs.
```

Comment:
```suggestion
    For generative mode to work, inputs from the graph and `LOOPBACK` must be decoupled, meaning `Process` can be called with complete set of inputs from the graph, but also with just `LOOPBACK`. It can be achieved via `SyncSetInputStreamHandler`. Above configuration sample creates a set with `LOOPBACK`, which also, implicitly creates another set with all remaining  inputs.
```